x-pack/plugin/esql/src/main/java/org/elasticsearch/xpack/esql/planner/PlannerUtils.java

```diff
@@ -211,6 +211,14 @@ public static ElementType toSortableElementType(DataType dataType) {
      * Map QL's {@link DataType} to the compute engine's {@link ElementType}.
      */
     public static ElementType toElementType(DataType dataType) {
+        // TODO: Remove this method and update all calls to pass the extra boolean
```

Comment:
With the latest design, the use of `forStats==true` is limited to the local physical plan, so most of the time it is completely correct to set this to false. Should we bother changing all the callers to this method when the case where it is needed is limited to a single location?